fw/http_msg.c

```diff
@@ -912,8 +912,9 @@ tfw_http_msg_hdr_xfrm_str(TfwHttpMsg *hm, const TfwStr *hdr, unsigned int hid,
 	}
 
 	if (unlikely(append && hid < TFW_HTTP_HDR_NONSINGULAR)) {
-		T_WARN("Appending to nonsingular header %d\n", hid);
-		return -ENOENT;
+		T_WARN("Appending to singular header '%.*s'\n",
+		       PR_TFW_STR(TFW_STR_CHUNK(hdr, 0)));
+		return 0;
```

Comment:
Why we don't drop response in this case now?

Reply:
We shouldn't, it is a legit case. Just print a warning.